### ISSUE TITLE
Updated  to 2.17.1, and get error

### ISSUE THREAD

**Issue by shouhuori** [expo/expo-cli#586]
## Description
after run `yarn start`,get this error:
```
Warning: Problem validating app.json: Unable to perform cache refresh for /Users/zhichou/.expo/cache/schema-32.0.0.json: XDLError: Invalid JSON returned from API: SyntaxError: Unexpected end of JSON input. Response body: .
Unable to read /Users/zhichou/.expo/cache/versions.json. SyntaxError: Unexpected token u in JSON at position 0
Error: Unable to read /Users/zhichou/.expo/cache/versions.json. SyntaxError: Unexpected token u in JSON at position 0
    at /xdl@53.7.1/src/tools/FsCache.js:101:15
    at Generator.next (<anonymous>)
    at step (/Users/zhichou/.config/yarn/global/node_modules/xdl/build/tools/FsCache.js:20:191)
    at /Users/zhichou/.config/yarn/global/node_modules/xdl/build/tools/FsCache.js:20:361
```
and `.expo/cache/versions.json` file content:
```
undefined
```
## Environment
MacOS: 10.14.4
node:v11.12.0
yarn:1.15.2
expo sdk :32.0.0

**Comment by shouhuori** [comment]
fixed,add `2.17.1`to `~/.expo/cache/verisons.json`

**Comment by salmanLee** [comment]
chinese? 兄弟，我在2.18.3版本遇到了同样的问题，我已经尝试了卸载expo-cli重新安装，但是问题依然存在，你的解决方法添加了2.17.1具体是什么东西呢，虽然看起来cache会被程序无限覆盖导致你的添加消失，但我还是希望解决眼下问题，谢谢

**Comment by InfiniteXyy** [comment]
I also encountered this problem, and when I deleted `export http_proxy....` in .~/zshrc. it works well.
Quite strange

**Comment by chenyu1990** [comment]
> I also encountered this problem, and when I deleted `export http_proxy....` in .~/zshrc. it works well.
> Quite strange

thx

**Comment by Damoness** [comment]
> I also encountered this problem, and when I deleted `export http_proxy....` in .~/zshrc. it works well.
> Quite strange

Thank you

**Comment by 234146326** [comment]
> fixed,add `2.17.1`to `~/.expo/cache/verisons.json`

> This problem fills in the version number, the problem still exists.

```
Starting project at /Users/jerryxu/wwwroot/cache/test/expo_projuct_1/expo_1
Expo DevTools is running at http://localhost:19002
Opening DevTools in the browser... (press shift-d to disable)
Unable to read /Users/jerryxu/.expo/cache/versions.json. SyntaxError: Unexpected token u in JSON at position 0
Error: Unable to read /Users/jerryxu/.expo/cache/versions.json. SyntaxError: Unexpected token u in JSON at position 0
    at Cacher.getAsync (/@expo/xdl@57.9.1/src/tools/FsCache.ts:109:15)
    at versionsAsync (/@expo/xdl@57.9.1/src/Versions.ts:58:10)
    at Object.sdkVersionsAsync (/@expo/xdl@57.9.1/src/Versions.ts:62:27)
    at _validateExpJsonAsync (/@expo/xdl@57.9.1/src/project/Doctor.ts:195:21)
    at validateAsync (/@expo/xdl@57.9.1/src/project/Doctor.ts:450:21)
    at startExpoServerAsync (/@expo/xdl@57.9.1/src/Project.ts:2078:9)
    at Object.startAsync (/@expo/xdl@57.9.1/src/Project.ts:2397:5)
    at action (/Users/jerryxu/.config/yarn/global/node_modules/expo-cli/src/commands/start.ts:163:3)
    at /Users/jerryxu/.config/yarn/global/node_modules/expo-cli/src/commands/start.ts:334:16
    at Command.<anonymous> (/Users/jerryxu/.config/yarn/global/node_modules/expo-cli/src/exp.ts:85:7)
```
